From d8bb8179be994542c83d83ab23714adb9601967c Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 30 Jul 2023 10:35:04 +0200
Subject: [PATCH 412/464] bes2600: Disable STANDARD_FACTORY_EFUSE_FLAG

Until we know what this does, disable it. It was not in the previous
version and required different factory data.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/staging/bes2600/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/bes2600/Makefile b/drivers/staging/bes2600/Makefile
index 82622c1dda1b..ebe11837cb98 100644
--- a/drivers/staging/bes2600/Makefile
+++ b/drivers/staging/bes2600/Makefile
@@ -111,7 +111,7 @@ endif
 ifeq ($(CONFIG_BES2600_CALIB_FROM_LINUX),y)
 FACTORY_CRC_CHECK                    ?= n
 FACTORY_SAVE_MULTI_PATH              ?= y
-STANDARD_FACTORY_EFUSE_FLAG          ?= y
+STANDARD_FACTORY_EFUSE_FLAG          ?= n
 ifeq ($(FACTORY_SAVE_MULTI_PATH),y)
 FACTORY_PATH                         ?= /data/cfg/bes2600_factory.txt
 FACTORY_DEFAULT_PATH                 ?= /lib/firmware/bes2600_factory.txt
-- 
2.34.1

